qiskit/extensions/quantum_initializer/uc.py

```diff
@@ -47,6 +47,7 @@
 from qiskit.circuit.exceptions import CircuitError
 from qiskit.exceptions import QiskitError
 from qiskit.quantum_info.synthesis import OneQubitEulerDecomposer
+from qiskit.extensions.quantum_initializer import _warn_if_bad_numpy
```

Comment:
I normally address lint's cyclic import complaints with relative imports, but I guess here you'd have to move `_warn_if_bad_numpy` to a separate file to be able to do that, right?